misc/net.minetest.minetest.desktop

```diff
@@ -7,7 +7,6 @@ Comment[fr]=Plate-forme de jeu multijoueurs à base de blocs
 Exec=luanti
 Icon=luanti
 Terminal=false
-PrefersNonDefaultGPU=true
```

Comment:
Please keep a short comment somewhere mentioning the issue with `PrefersNonDefaultGPU`. Depending on how long that switcheroo-control merge takes, another contributor might be inclined to re-add the line if it somehow works for them. Let's let them know that there's upstream issues that need to be addressed first.